lib/from_erlang.ml

```diff
@@ -31,13 +39,26 @@ let rec expr_of_erlang_expr = function
     | F.ClsCase (_, _, _, _) | F.ClsFun (_, _, _, _) -> failwith "not implemented"
     ) in
     Case (expr_of_erlang_expr e, cs)
+  | ExprLocalFunRef (_line_t, name, arity) ->
+     (* TODO: support local `fun F/A` *)
+     failwith "not implemented"
```

Comment:
Please make a issue.

Reply:
and
```ocaml
raise Known_error.(FialyzerError (NotImplemented issue))
```

Reply:
OK. I'll do that.

Reply:
https://github.com/dwango/fialyzer/pull/78/commits/0e84ef82149c15a23a3da4203d5092029db76693